.build/images/Quartz64/quartz64_defconfig

```diff
@@ -1122,3 +1134,16 @@ CONFIG_ZSWAP=y
 CONFIG_ZSWAP_COMPRESSOR_DEFAULT_ZSTD=y
 CONFIG_ZSWAP_DEFAULT_ON=y
 CONFIG_ZSWAP_ZPOOL_DEFAULT_ZSMALLOC=y
+CONFIG_RT_GROUP_SCHED=y
```

Comment:
Same as `CONFIG_BLK_DEV_THROTTLING`, this group scheduler is not available on RPi or Debian kernel either, so seems to be a very rare use case. Are you sure if is needed for Calico networks?